internal/xds/bootstrap.go

```diff
@@ -79,11 +79,11 @@ func SetupBootstrapFile(opts BootstrapOptions) (func(), error) {
 	}
 	logger.Infof("Created bootstrap file at %q with contents: %s\n", f.Name(), bootstrapContents)
 
-	origBootstrapFileName := env.BootstrapFileName
-	env.BootstrapFileName = f.Name()
+	origBootstrapFileName := envconfig.BootstrapFileName
```

Comment:
Hmm, most (all?) of the xds variable names don't have "xds" in it. Would that be confusing?

Reply:
Makes sense. I tweaked the variable names a bit; PTAL